compatibility-verifier/compCheck.sh

```diff
@@ -112,14 +122,18 @@ function stopService() {
 # Starts a Pinot cluster given a specific target directory
 function startServices() {
   dirName=$1
+  local controllerConfigFile=$2
```

Comment:
All config files are supposed to be in the config/ directory, right? So, we can just pass one arg here?

Reply:
passed config folder to startServices